standard-deployment.cfg

```diff
@@ -18,6 +19,7 @@ parts -=
 
 usernamelogger_ac_cookie_name = __ac
 raven_project_dist = opengever.core
+slack-webhook = $STANDARD_SLACK_WEBHOOK
```

Comment:
Yes you are correct, but that it is actually the intent:
The variable goes directly into the shell script: https://github.com/4teamwork/ftw-buildouts/blob/master/slacker.cfg#L14
The shell script will then contain:
```
URL="$STANDARD_SLACK_WEBHOOK"
```

This will read the environment variable at execution time.
I have set this up and tested it, looks like it works well.

The variable is meant to be set in `/home/zope/etc/env.d/slacker.sh` and should not be exposed.